framework/packages/abstract-interface/src/ibc.rs

```diff
@@ -62,3 +60,61 @@ impl<Chain: CwEnv> AbstractIbc<Chain> {
         ])
     }
 }
+
+#[cfg(feature = "interchain")]
+// Helpers to create connection with another chain
+pub mod connection {
+    use super::*;
+    use abstract_std::account_factory::ExecuteMsgFns;
+    use abstract_std::ibc_client::ExecuteMsgFns as _;
+    use abstract_std::ibc_client::QueryMsgFns;
+    use abstract_std::ibc_host::ExecuteMsgFns as _;
+    use abstract_std::objects::chain_name::ChainName;
+    use cw_orch_interchain::prelude::*;
+    use cw_orch_interchain::InterchainError;
+    use cw_orch_polytone::Polytone;
+    /// This is used for creating a testing connection between two Abstract connections using an existing polytone connection
+    ///
+    /// You usually don't need this function on actual networks if you're not an Abstract maintainer
+    pub fn abstract_ibc_connection_with<Chain: IbcQueryHandler, IBC: InterchainEnv<Chain>>(
```

Comment:
This should connect both ways.

Reply:
Couldn't make it work, got `HostAddressExists` when trace is longer than 1

Reply:
So there is `test_multi_hop_account_creation` test where I managed to connect in both ways, (polytone1-polytone-2 and polytone2-polytone-1) but then it's not possible to connect polytone2 to polytone3. Let's move this to separate task, so we don't dive too hard here

Reply:
Ok, I now understand why this is failing and attempting to fix that now.

Reply:
> Ok, I now understand why this is failing and attempting to fix that now.

Is it related to this PR?

Reply:
Yes, this is regarding your `test_multi_hop_account_creation` test. This will be fixed in a future PR, with better handling of the polytone complexity. See internal comms for suggestions